### PR TITLE
fix: allow missing description on taak completion

### PR DESCRIPTION
**Pull request by hsiliev**
Allow missing (null) description on completing a taak with send document.

Solves: PZ-3077

### CODE REVIEW

**Review by edgarvonk — APPROVED** [review]
looks good! could maybe add a unit test for this? no high prio though

**Comment by hsiliev** [comment]
> looks good! could maybe add a unit test for this? no high prio though

Extended the existing test to cover the `TAAK_DATA_DOCUMENTEN_VERZENDEN_POST` branch

File: src/main/kotlin/net/atos/zac/app/taken/TakenRESTService.kt [hunk 1]
```diff
@@ -218,9 +218,9 @@ class TakenRESTService @Inject constructor(
             taakdata[TaakVariabelenService.TAAK_DATA_DOCUMENTEN_VERZENDEN_POST]?.let {
                 updateVerzenddatumEnkelvoudigInformatieObjecten(
                     documenten = it,
-                    // implicitly assume that the following keys are present in taakdata
+                    // implicitly assume that the verzenddatum key is present in taakdata
                     verzenddatumString = taakdata[TaakVariabelenService.TAAK_DATA_VERZENDDATUM]!!,
-                    toelichting = taakdata[TaakVariabelenService.TAAK_DATA_TOELICHTING]!!
+                    toelichting = taakdata[TaakVariabelenService.TAAK_DATA_TOELICHTING]
```
Comment (bas-info-nl) [inline src/main/kotlin/net/atos/zac/app/taken/TakenRESTService.kt:223]: Is there no default reason that should be used if none is given?

**Review by bas-info-nl — APPROVED** [review]

File: src/main/kotlin/net/atos/zac/app/taken/TakenRESTService.kt (hunk 1 above)
Comment (hsiliev) [reply src/main/kotlin/net/atos/zac/app/taken/TakenRESTService.kt:223]: @camielb @edgarvonk Do we have a default reason for completing a task? If not should we have one? WDYT?

Comment (edgarvonk) [reply src/main/kotlin/net/atos/zac/app/taken/TakenRESTService.kt:223]: @hsiliev I have no idea. I think that is something to check with our PO?

**Pull request merged**